tests/conftest.py

```diff
@@ -1,11 +1,31 @@
 import asyncio
+import uuid
+from distutils.version import StrictVersion
+from os import environ as ENV
 
 import pytest
 
 from aiodocker.docker import Docker
 from aiodocker.exceptions import DockerError
 
 
+_api_versions = {
```

Comment:
would be nice also to test against different version of Docker/API.

What do you think?